react_juce/core/View.cpp

```diff
@@ -9,6 +9,8 @@
 
 #include "View.h"
 
+#include <climits>
```

Comment:
@nick-thompson, build fix I required on Linux. I think you added this to your quickjs branch too

Reply:
Yep :+1:. I'll address mine when I rebase the qjs branch onto this